### PR TITLE
Rajendra/app id cont execution

### PR DESCRIPTION
**Pull request by rajendra-deriv**
PR for performing Continuous Execution

### CODE REVIEW

**Review by markodonnell-deriv — APPROVED** [review]
Looks good

**Review by abuzar-deriv — COMMENTED** [review]
Changes look good; but looks like this cypress/support/helper/applicationRegister.js file is not formatted since Prettier is throwing error. Which makes me wonder, does it not run for you on commit, @rajendra-deriv?

**Comment by rajendra-deriv** [comment]
> Changes look good; but looks like this cypress/support/helper/applicationRegister.js file is not formatted since Prettier is throwing error. Which makes me wonder, does it not run for you on commit, @rajendra-deriv?

No @abuzar-deriv , dont know why Prettier is not running for me on commit.

**Review by abuzar-deriv — APPROVED** [review]
LGTM

**Review by bilalmajid-deriv — APPROVED** [review]
LGTM

File: cypress/e2e/api/createApplicationRegister.cy.js [hunk 1]
```diff
@@ -1,29 +1,24 @@
...
-let updatedAppId
 describe('QATEST - 148419 - Register a New Application / App ID', () => {
-  it('Creation of New App ID should be successful. ', () => {
-    // Login is required as we need Auth Token for running RegisterApplication API.
-    cy.c_login() 
-    const oldAppdId = parseInt(Cypress.env('configAppId'))
-    cy.task('wsConnect')
-    cy.c_authorizeCall()
+  beforeEach(() => {
+    cy.log('<prevAppId - beforeEach> ' + Cypress.env('setupComplete'))
```
Comment (meeyun-deriv) [inline cypress/e2e/api/createApplicationRegister.cy.js:5]: is this needed to put into `cypress.config.js`?

**Review by meeyun-deriv — COMMENTED** [review]
Can you please share the default value of `E2E_RUN_FROM_PR` and `setupComplete` to me through lastpass?

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Pull request merged**